### DISCUSSION TITLE
Revalidation behaviour of fetcher#load vs. fetcher#submit

### DISCUSSION THREAD

**Discussion by ericchernuka** [remix-run/remix#9658] [Q&A]
We're currently using Remix for client-side data fetching using `useFetcher`. We typically have areas that we'd like to load the data once (modal, one time call to action) etc, but are wondering if when using `fetcher.load` or `fetcher.submit(null, { method: "GET" })` are supposed to be forever subscribed to revalidations when actions are performed. We had thought the point of this would be or an imperative one-time data load, but now we're getting 10's of revalidations that are unexpected.

I've reproduced this in a [Stackblitz](https://stackblitz.com/edit/remix-run-remix-pjfwvz) to show it occurs, whether this is intentional, and whether there's any way around this without having to bail out to raw `fetch`.

**Comment by kiliman** [comment]
For API/resource routes, I would just stick with raw `fetch`.

It would be nice if you could add a `revalidate: false` option for fetchers.
